src/conn/conn_layered.c

```diff
@@ -311,7 +311,7 @@ __wt_layered_table_manager_start(WT_SESSION_IMPL *session)
     WT_ERR(__wt_calloc(session, sizeof(WT_LAYERED_TABLE_MANAGER_ENTRY *),
       manager->open_layered_table_count, &manager->entries));
     manager->entries_allocated =
-      manager->open_layered_table_count * sizeof(WT_LAYERED_TABLE_MANAGER_ENTRY);
+      manager->open_layered_table_count * sizeof(WT_LAYERED_TABLE_MANAGER_ENTRY *);
```

Comment:
Should this just be `open_layered_table_count` without the multiplication?

Reply:
No, this is the actual bytes being allocated.